task1/Car.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task1
+{
+    class Car : Vehicle
+    {
+        public int DoorsCount { get; set; }
```

Comment:
The same as for bicycle. The state of car can't be changed I believe.
Lets leave getters only.